scripts/.gitignore

```diff
@@ -0,0 +1 @@
+**/node_modules
```

Comment:
Do we want to include build also here? @IsAmrish

Reply:
there is not `build` script, so no need to add